.buildkite/ftr_oblt_stateful_configs.yml

```diff
@@ -24,7 +24,6 @@ enabled:
   - x-pack/test/api_integration/apis/logs_ui/config.ts
   - x-pack/test/api_integration/apis/logs_shared/config.ts
   - x-pack/test/api_integration/apis/metrics_ui/config.ts
-  - x-pack/test/api_integration/apis/osquery/config.ts
```

Comment:
I think it was added accidentally, config is listed in `.buildkite/ftr_platform_stateful_configs.yml`

Reply:
Found some others that are enabled in both `oblt` and `platform`:
- x-pack/test/api_integration/apis/logs_ui/config.ts
- x-pack/test/api_integration/apis/metrics_ui/config.ts
- x-pack/test/api_integration/apis/osquery/config.ts
- x-pack/test/api_integration/apis/synthetics/config.ts

Reply:
Thank you, I raised https://github.com/elastic/kibana/pull/219089 to fix + check in future